src/main/java/seedu/address/logic/parser/tags/AddTagCommandParser.java

```diff
@@ -0,0 +1,64 @@
+package seedu.address.logic.parser.tags;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_INDEX;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.tags.AddTagCommand;
+import seedu.address.logic.parser.ArgumentMultimap;
+import seedu.address.logic.parser.ArgumentTokenizer;
+import seedu.address.logic.parser.Parser;
+import seedu.address.logic.parser.ParserUtil;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.tag.Tag;
+
+/**
+ * Parses input arguments and creates a new AddTagCommand object.
+ */
+public class AddTagCommandParser implements Parser<AddTagCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of an AddTagCommand.
+     * Looks for the name of the added tag under PREFIX_NAME, and looks for indices and tag parameters.
+     * There must be at least one index or sub-tag specified.
+     * @throws ParseException if the user input does not conform to the expected format.
```

Comment:
from what i've seen from the other classes, i believe this line should not end with a full stop